Microsoft.Toolkit.Uwp.Connectivity/BluetoothLEHelper/BluetoothLEHelper.cs

```diff
@@ -59,11 +59,15 @@ public class BluetoothLEHelper
         /// </summary>
         private BluetoothAdapter _adapter;
 
+        private DispatcherQueue _dispatcherQueue;
+
         /// <summary>
         /// Prevents a default instance of the <see cref="BluetoothLEHelper" /> class from being created.
         /// </summary>
         private BluetoothLEHelper()
         {
+            _dispatcherQueue = DispatcherQueue.GetForCurrentThread();
```

Comment:
>GetForCurrentThread [](start = 47, length = 19)

What if someone creates a BluetoothLEHelper on a background thread?

Reply:
We should call this out on the documentation. Is there anyway we can fix this? Maybe allow change from a private field to a public settable property would make more sense. This was they have more control over which DispatcherQueue will be used, even if they create it from a background thread, as long as they have access to a valid DispatcherQueue.

Reply:
I've changed them so they can be set after creation as well. We'll call it out on the docs.